rasa_core/processor.py

```diff
@@ -29,11 +29,17 @@
 from rasa_core.tracker_store import TrackerStore
 from rasa_core.trackers import DialogueStateTracker
 
-scheduler = BackgroundScheduler()
-scheduler.start()
+
 
 logger = logging.getLogger(__name__)
 
+try:
+    scheduler = BackgroundScheduler()
+    scheduler.start()
+except:
```

Comment:
can we catch a more specific exception here? or is it just throwing a generic `Exception`?